run

```diff
@@ -22,25 +26,58 @@ parser.add_argument(
     action="store_true",
     help="Run container in background & don't show container logs",
 )
+# Set to false if your current user is already in the docker user group
+parser.add_argument(
+    "--sudo",
+    default=False,
+    action="store_true",
+    help="Run commands with sudo",
+)
 args = parser.parse_args()
 
+use_sudo = args.sudo
+if not args.sudo:
+    current_user = getpass.getuser()
+    user_in_docker_group = current_user in grp.getgrnam("docker").gr_mem
```

Comment:
This feels like it should be in its own function (`is_user_in_group()` or similar)

Reply:
done